assets/js/collection.js

```diff
@@ -5,25 +5,25 @@ storefrontApp.controller('collectionController', ['$scope', '$location',  functi
     var $ctrl = this;
     $ctrl.init = function() {
         $ctrl.sortModes = {
-            'manual': 'collections.sorting.featured',
-            'best-selling': 'collections.sorting.best_selling',
-            'title-ascending': 'collections.sorting.az',
-            'title-descending': 'collections.sorting.za',
+            'priority-descending;name-ascending': 'collections.sorting.featured',
+            // 'best-selling': 'collections.sorting.best_selling',
+            'name-ascending': 'collections.sorting.az',
+            'name-descending': 'collections.sorting.za',
             'price-ascending': 'collections.sorting.price_ascending',
```

Comment:
Why price filter is not modified here as in Electronics theme? Is it working that way?

Reply:
It's working because we don't have multicurrency in b2b store
![image](https://user-images.githubusercontent.com/22875828/67464409-96843800-f643-11e9-88af-771a929f88de.png)
![image](https://user-images.githubusercontent.com/22875828/67464340-781e3c80-f643-11e9-85cc-69e2d624ca6f.png)